### ISSUE TITLE
httk R package 

### ISSUE THREAD

**Issue by ElenaDR1** [VHP4Safety/cloud#29]
Please fill out the below form as complete as possible, but empty fields can be filled out later. 

# httk R package

The httk R package is an open access tool which provides access to different types of pharmacokinetic (PK) and PBPK models, parameter values and experimental data (Pearce et al. 2017, Silva and Kwok 2022). 
This tool was explained in the Model group document ("Proposals for Modelling in VHP4Safety". The PBPK model group (WP1.2) considers the integration of httk in the platform may be useful. 

* Does the service require a login? NO
* Technology Readiness Level (TRL)?
* API Type? REST | SOAP | other
* Who are the targeted users?
* Relevant VHP4Safety Case Study?

## Specification

* Provided by? WHO IS PROVIDING THE SERVICE? R
* Citation? doi: 10.18637/jss.v079.i04 ; https://doi.org/10.1016/j.crtox.2022.100064
* Version?
* License?
* Source Code?
* Docker?
* ELIXIR Bio.tools?
* ELIXIR TeSS?

**Comment by ozancinar** [comment]
It's a little ambiguous how we can add an R package to the catalog. Surely, we can add an item to the package's page on CRAN but its utilization could be vague. Instead, we could look into deploying an R/RStudio instance where the httk package is preinstalled.

**Comment by egonw** [comment]
> It's a little ambiguous how we can add an R package to the catalog.

Yeah, agreed. The same for how to deal with the 'data' services. The JSON does not have a field for this kind of annotation.

@ozancinar, @Maddocent, do you see a reason why we should not just add a field for that? Maybe with a value from an enumerated list, which we could start like:

- online tool
- standalone tool (local installation)
- package (local install into platform like R) 
- data or database

**Comment by ozancinar** [comment]
Last comment -- This will not go on, I presume. I'm not going to close it but feel free to do so.
